sirepo/pkcli/service.py

```diff
@@ -52,25 +59,159 @@ def flower():
         ])
 
 
-def http():
-    """Starts Flask server in http mode.
+def flask():
+    from sirepo import server
+
+    use_reloader = pkconfig.channel_in('dev')
+    app = server.init(use_reloader=use_reloader)
+    # avoid WARNING: Do not use the development server in a production environment.
+    app.env = 'development'
+    app.run(
+        host=cfg.ip,
+        port=cfg.port,
+        threaded=True,
+        use_reloader=use_reloader,
+    )
+
+
+def http(driver='local', nersc_proxy=None, nersc_user=None, sbatch_host=None):
```

Comment:
I guess I wasn't clear. The only drive we want is "local". It's the only one that can work in this simplistic mode. This is for the curl installer and for external developers. They won't be dealing with sbatch or docker configuration.

Reply:
Argh sorry I missed that. Will do.